IntelligentRoboticsLab/dnt-hulk
# Longer loser role duration

## Introduced Changes

Adds an extra 10s timer for the loser role

## How to Test

Run robot, make it striker by letting it see the ball. Remove ball and see the role change from striker to loser. After 10 seconds of not finding the ball the role should change to searcher.
